searx/templates/__common__/about.html

```diff
@@ -80,9 +79,9 @@ <h2>Where to find anonymous usage statistics of this instance ?</h2>
   <h2>How can I make it my own?</h2>
 
   <p>
-    Searx appreciates your concern regarding logs, so take the
-    code from the <a href="https://github.com/searx/searx">original searx project</a> and
-    run it yourself!
+    Searx appreciates your concern regarding logs, so take the code from
```

Comment:
searx -> searxng

Reply:
I replaced searx by SearXNG in the complete _about_ side. https://github.com/searxng/searxng/pull/25/commits/abfabebd9173086d815c19e34212f1f24676c5ee#diff-7dc09c2bec99a1049f972f8d2d5f19db05b8ee5064452036c232f995eac3a603